gen/host.go

```diff
@@ -356,6 +356,10 @@ func genPluginMethod(g *protogen.GeneratedFile, f *fileInfo, method *protogen.Me
 				resSize &^= %s
 			}
 
+			if resPtr != 0 {
```

Comment:
there is a comment in the generated code above this that literally says

```
// Note: This pointer is still owned by TinyGo, so don't try to free it
```

So, we need to remove that comment, and above this line add a new one like this:
```suggestion
			// We don't need the memory after deserialization: make sure it is freed.
			if resPtr != 0 {
```